lms/djangoapps/instructor/views/legacy.py

```diff
@@ -245,19 +245,31 @@ def domatch(student):
         if not aname:
             msg += "<font color='red'>{text}</font>".format(text=_("Please enter an assignment name"))
         else:
-            allgrades = get_student_grade_summary_data(request, course, get_grades=True, use_offline=use_offline)
+            allgrades = get_student_grade_summary_data(
+                request,
+                course,
+                get_grades=True,
+                use_offline=use_offline,
+                get_raw_scores=True,
```

Comment:
This is wrong. We don't want to `get_raw_scores` for the remote gradebook. We want assignments.